modules/gapi/src/backends/cpu/gcpustereo.cpp

```diff
@@ -0,0 +1,48 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include <opencv2/gapi/stereo.hpp>
+#include <opencv2/gapi/cpu/stereo.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+
+#include <opencv2/calib3d.hpp>
+
+GAPI_OCV_KERNEL_ST(GCPUStereo, cv::gapi::ster::GStereo, cv::gapi::ster::StereoSetup)
```

Comment:
`StereoSetup` may be just a structure to initialize state and state may be just `StereoBM` class, don't you think?

Reply:
We also have to keep `focus` and `baseLine` to calculate depths from disparity 
Easier way  to keep it all in `StereoSetup` structure, I think...
Or what do you mean?

Reply:
Please split the `StereoSetup` structure from `cv::StereoBM`. Lets `cv::StereoBM` be implementation details and be hidden from user. Lets `cv::StereoBM` be the state and `StereoSetup` be the initialization structure.

Reply:
If you can't retrieve `focus` and `baseline` from the `cv::StereoBM`, then create a structure to hold `cv::StereoBM` and `focus` + `baseline`. Lets this structure be the state and `StereoSetup` be the structure to just initialize the state.

Reply:
Done